.github/workflows/overall-tests.yml

```diff
@@ -0,0 +1,59 @@
+name: Overall Test Workflow
+
+on:
+  pull_request_review:
+    types: [submitted]
```

Comment:
For example, if JR approves a PR, and Fabien approves it right after, does it mean the action will be run twice?
Or it will only be run once because there is no change between the 2 approves?